conversion/converter/command_converter.py

```diff
@@ -9,16 +9,34 @@
 from qiskit.circuit.library.standard_gates import *
 import pyquil.quilbase as pyquil_circuit_library
 from pyquil.gates import NOP, MEASURE
+import cirq
+from cirq import Circuit
+from braket.circuits.circuit import Circuit as BRCircuit
 
-def pyquil_commands_to_program(commands: str) -> QuantumCircuit:    
+
+def pyquil_commands_to_program(commands: str) -> Program:
     exec(commands)
     val = eval("p")
```

Comment:
From Martin's comments: "the use of eval and exec is not safe (is there an alternative solution?)"